product-service/serverless.yml

```diff
@@ -78,8 +135,18 @@ functions:
           path: products
           method: post
           cors: true
-  
 
+  catalogBatchProcess:
+    handler: dist/catalogBatchProcess.handler
+    url: true
+    events:
+      - sqs:
+          arn: "arn:aws:sqs:eu-west-1:009688690949:import-service-queue"
```

Comment:
Same
As we implement automation in creating instance it is better to avoid hardcoded refs like this. You can use import/export through services. In that case you avoid side effects if for any reason this link will change